app/helpers/application_helper.rb

```diff
@@ -43,6 +43,11 @@ def back_link(path = :back, text: "Back", **options)
   end
 
   def link_to_change_answer(step)
-    link_to "Change", teacher_training_adviser_step_path(step.key), { class: "govuk-link" }
+    link_to(teacher_training_adviser_step_path(step.key), { class: "govuk-link" }) do
+      safe_html_format("Change <span class='visually-hidden'> #{step.key.humanize(capitalize: false)}</span>")
```

Comment:
I think some of these step names need changing as the file names aren't that user friendly (like the 'identity' step)